### DISCUSSION TITLE
Receber no exterior de conta PJ direto pra conta PF

### DISCUSSION THREAD

**Discussion by glauberramos** [DyegoCosta/trabalhando-remoto#192] [General]
Tendo uma conta PJ no exterior como o Payoneer por exemplo, posso repassar os valores recebidos nessa conta para uma conta PF do exterior (como dividendo sem passar pelo brasil, sendo a empresa brasileira)?

Minha dúvida é se isso é de acordo com a lei sendo que vou declarar os valores no brasil normalmente. A minha intenção é evitar transformar em reais, transferir pra conta PF no brasil e fazer remessa para uma conta no exterior de novo, pagando taxas duas vezes.

**Comment by shamanime** [comment]
As isenções de imposto pra serviços prestados pro exterior só se aplicam quando o dinheiro passa para o Brasil.

É bom falar com com um contador pra mais informações.

**Comment by bolds07** [comment]
Eu acho engraçado a turma pagar contador e vir num forum de internet perguntar as coisas que o contador devia fazer

**Comment by glauberramos** [comment]
@bolds07 a troca de experiencia é sempre válida, ainda mais com pessoas na mesma situação que a sua, eu tbm entrei em contato com o contador para esclarecer sobre isso. Forums na internet são feitos exatamente para isso, tirar dúvidas e discutir e compartilhar.

**Comment by bolds07** [comment]
Com certeza todo mundo pode se ajudar 
O importante é entender que se você está pagando alguém pra resolver essas broncas (contador)  e mesmo assim está recorrendo a Internet,  talvez esse profissional não esteja fazendo jus ao pagamento 


Sobre a sua pergunta eu diria que não é possível.

Toda receita de Pf em moeda estrangeira tem que ser declarada e tem inclusive um programa específico pra isso 

Mas é aquilo eu não faço idéia

Na minha cabeça é muito mais simples você manter na conta PJ em dólar e movimentar normalmente usando o cartão para pagar suas contas pessoais

**Comment by igorlvicente** [comment]
> Eu acho engraçado a turma pagar contador e vir num forum de internet perguntar as coisas que o contador devia fazer

Como saber se o contador está fazendo errado se você não sabe o que é o certo? Acho que por esse problema, vale a pena discutir aqui sobre o assunto.

**Comment by bolds07** [comment]
> Como saber se o contador está fazendo errado se você não sabe o que é o certo? 

Em teoria você "confia" neh
Isso é uma discussão muito mais longa que eu já tive sobre pra que pagar contador (já que não existe nenhuma obrigação legal)

Você não vai no médico e depois joga no Google o nome dos remédios pra saber se aqueles remédios realmente servem e se a dosajem está correta (quando eu quero procurar informação no Google eu não perco meu tempo/dinheiro pagando um profissional depois para me tirar a mesma dúvida).

Você não chama o seguro pra trocar o pneu do carro e fica do lado olhando no Google pra saber se ele está fazendo o certo mesmo neh.

**Comment by glauberramos** [comment]
@bolds07 cara eu faço exatamente isso que você esta falando para não fazer, eu nunca confio 100% em um profissional que eu to contratando, eu sempre fico com um pé atrás e pesquiso na internet. Pode ser sobre remédios, coisas sobre minha empresa, produtos, etc...

A internet na minha opinião é a maior invenção do século 20, e eu acho super arriscado tu confiar alguma coisa em apenas 1 profissional, ainda mais um profissional que nem sabe usar o google direito as vezes, porque pesquisar bem no google é muito mais dificil do que parece.

Mas já estamos mudando bastante do assunto original da thread. Abs

**Comment by RangelReale** [comment]
Pelo que entendi do [site da contabilizei](https://www.contabilizei.com.br/contabilidade-online/quanto-minha-empresa-de-servicos-paga-de-impostos-no-exterior/), parece ser isso mesmo:

```
prestação de serviços para pessoa física ou jurídica residente ou domiciliada no exterior, cujo pagamento represente ingresso de divisas
```

abri um chamado com eles para confirmar.

**Comment by murillobit** [comment]
@RangelReale @glauberramos vocês chegaram a alguma conclusão sobre esse assunto? 

Estou na mesma situação, a contadora me orientou a transferir o dinheiro entre as contas PJ e PF internacionais e lançar como distribuição de lucro, porém ainda não ficou claro para mim se na minha PF internacional esse dinheiro vai ser tributado ou se o tratamento é o mesmo de quando o dinheiro é nacionalizado.

**Comment by RangelReale** [comment]
@murillobit segundo a contabilizei, a conta PJ em dolar da BS2 já vale como conta PJ, não precisa converter para reais nem mandar para uma conta PF em dolar antes. Eu estou mandando de lá direto para a Interactive Brokers.

**Comment by glauberramos** [comment]
@murillobit eu estou atualmente transferindo da Payoneer direto pra Interactive Brokers. O contador falou que é possível.

**Reply by mcferdev** [reply]
> @murillobit eu estou atualmente transferindo da Payoneer direto pra Interactive Brokers. O contador falou que é possível.

Opa....

Eu tentei transferir da Payoneer para 2 corretoras e nao deixaram por ser PJ a conta da payoneer. A sua conta é PJ e conseguiu transferir pra IB ?
Tentei o AvaTrade e o Etoro. Se conseguir pra IB assim, me interessa demais.
Outra coisa. Qual meio voce usa la? Bank Wire, Direct ACH Transfer from your Bank, Connect Your Bank via ACH, Online Bill Pay ou Mail a Check.

Abs, Marcelo.

**Comment by HarnonA** [comment]
Duas dúvidas:
1 - vocês mantêm valores em conta PJ no exterior?
2 - como transferem da conta PJ pra PF também no exterior?

Contador me passou que esses valores teriam que vir para o Brasil, não podia ficar no exterior.
E se eu quisesse mandar pra PF no exterior, teria que mandar da minha PF do Brasil.

@murillobit mencionou que dá pra distribuir dividendo e pagar carnê-leão, ja chegou a fazer a operação?

**Reply by murillobit** [reply]
@HarnonA 

1 - Eu não mantenho valores na conta PJ internacional por mais de um mês, o que faço são no máximo 3 transferências parciais dentro do mês para a conta PJ nacional, dependendo do movimento do câmbio. Faço isso por opção, não sei dizer se manter por mais tempo é permitido ou não... Mas acho que de fato a parte contábil fica um pouco mais complexa, então talvez eles prefiram dizer que não é permitido.
2 - Eu fiz isso só uma vez, pq colocando na ponta do lápis não valia tanto o custo-benefício o meu caso, por causa das tarifas do meu banco (BS2) para fazer transferência internacional. 

Sobre a questão de dividendo/carnê-leão que vc disse, na verdade eu não tinha certeza também não, estava justamente querendo descobrir isso...

Resumindo que eu descobri sobre esse assunto depois do meu último comentário, é possível sim e não é necessário pagamento de carnê leão, visto que a empresa é Brasileira apesar da conta ser internacional. Para fazer essa operação, é necessário que vc declare na nota fiscal o valor total do serviço, mesmo que vc não nacionalize todo o dinheiro. Dessa forma, vc paga todos os impostos já na sua PJ.

Vou deixar aqui os recursos que foram minhas principais fontes de informação:
- Contador: [Contabilizei](https://www.contabilizei.com.br/programa-de-indicacao?ref=cbbff11326b36c6d45f44bab46bbe2dc&nome=Murillo&email=murillobittencourt@gmail.com&utm_source=plataforma&utm_campaign=MGM&utm_source=plataforma).
- Advogado tributarista: fiz uma consultoria com mais 3 colegas de trabalho (ajudou a reduzir o custo) com o [Guilherme da Capital Global](https://capitalglobal.com.br/). Além de tirar nossas dúvidas, ele apresentou várias opções de configuração empresarial para quem atua no exterior.
- Artigos sobre o tema após pesquisa no Google: apesar de ser bem difícil achar informação de qualidade e confiável, o melhor artigo que achei foi [esse](https://bueno.tax/artigos/2021/01/distribuicao-de-dividendos-sem-cobertura-cambial-aspectos-regulatorios-e-tributarios-da-transferencia-de-valores-detidos-no-exterior-por-pessoa-juridica-brasileira-para-conta-bancaria-estrangeira-de/).

**Reply by bolds07** [reply]
> Duas dúvidas: 1 - vocês mantêm valores em conta PJ no exterior? 2 - como transferem da conta PJ pra PF também no exterior?
> 
> Contador me passou que esses valores teriam que vir para o Brasil, não podia ficar no exterior. E se eu quisesse mandar pra PF no exterior, teria que mandar da minha PF do Brasil.
> 
> @murillobit mencionou que dá pra distribuir dividendo e pagar carnê-leão, ja chegou a fazer a operação?

Já disse uma vez e digo novamente....
A maioria dos contadores que tem ai é muito incompetente e não sabe fazer nada diferente do trivial, na verdade isso se aplica a um 80% do mercado de trabalho... pessoas são treinadas para apertar sempre o mesmo botão, preencher o mesmos campos dentro do sistema... se você pedir qualquer coisa que foge um pouquinho do modelo que eles estão acostumados eles não sabem fazer... e pior, não sabem nem onde buscar a informação sobre como fazer

"Contador me passou que esses valores teriam que vir para o Brasil, não podia ficar no exterior"  nunca vi tamanha asneira... ai vai me dizer que vale, petrobras, gol, tam e todas as empresas exportadoras do brasil que mantem caixa em moeda estrangeira estão fora da lei... existe uma coisa chamada contabilidade hedge que é quando uma empresa com atuação em muitos paises, ao inves de converter todas as receitas para real, mantem as divisas internacionais e usa para pagar as despesas em moeda estrangeira [que é o minimo do bom senso]

até 2013-14, a petrobras convertia toda receita recebida em dolar para real no caixa, e depois precisava usar o real para comprar dolar para pagar os fornecedores estrangeiros... não precisa ser muito inteligente pra perceber que isso é burrada

por essas e por outras desinformações que ha mais de 2 anos eu opero sem contador... a maioria que tem ai seja online ou presencial so sabe preencher a pagina do simples e pronto... garanto que se o simples parar de calcular o imposto devido automaticamente e os contadores precisarem usar excel para fazer os calculos vai ter um monte de empresa sem saber como começar a conta

a sim, isso é outra coisa que eles nao te contam né... que se voce é micro empresario de serviços enquadrado no simples, voce nao é obrigado a ter contador... voce mesmo pode manter seu livro caixa  e fazer as declarações

**Reply by HarnonA** [reply]
Nao achei razoavel a explicação do contador, por isso decidi perguntar aqui.
Na minha cabeça, o que o estado quer é a parte dele, então bastaria declarar e pagar o imposto, não importaria onde o valor está. O que @murillobit falou meio que confirma isso. Mas tbm entendo que existem muitas normas e nem sempre são claras.

Já deu uma clareada nas ideias, valeu pessoal.

**Comment by bolds07** [comment]
Em teoria você está errado, mas na prática está certo
Essa idéia de que "o que o estado quer é que você pague imposto"  é
equivocada, porque para o estado se o imposto que você está pagando é fruto
de crime ou atividade ilícita não vale de nada

Então é como você falou tem muitas normas e processos para tentar garantir
que isso não ocorra

Maaaaaaas no Brasil,  na prática, com toda a competência máxima de nossas
instituições de poder público...  A menos que você esteja fazendo algo
muito escroto,  se você pagar o imposto certinho ninguém vai te incomodar

A prova da ingerência do nosso sistema público é simples, trabalhei na
petrobras de 2012 a 2016 e desde então nunca mais tive emprego publico,
essa semana eu recebi um email da CGU (imagine o susto) falando que eu
preciso fazer uma declaração de patrimônio que é devida a todos
funcionários públicos depois que foi criada uma nova lei em 2020

6 anos desde que eu pedi demissão e ainda ronda em algum sistema de dados
do governo informação como se eu fosse funcionário público

Mostrei o email a um amigo advogado e ele falou logo, "vamos processar a
petro por violar  a lgpd, ou ainda conseguir um registro desse sistema e
pedir configuração de vínculo empregaticio e 6 anos de salário atrasado"



Em sáb., 16 de jul. de 2022 12:10, Harnon Anizio ***@***.***>
escreveu:

**Comment by raulfragoso** [comment]
Somente para compartilhar, descobri que é possível enviar recursos da conta PJ da Payoneer (no meu caso dólares australianos) para uma conta PF de stockbroker nos USA, ao menos funciona com a TD Ameritrade.

Tenho feito isso desde o mês passado para investir parte do que recebo em AUD  nos stocks e REITs americanos - o que já vinha fazendo com a Avenue, mas eu era obrigado a primeiro nacionalizar os recursos, pagando impostos sobre a NF de serviços, e depois pagando ainda mais spread e IOF ao enviar os valores em USD para os USA.

Mandando direto da Payoneer para a broker, consigo evitar a nacionalização dos recursos e, consequentemente, os impostos de faturamento da empresa e spread em duplicidade.

Obviamente isso só vale para quem pretende investir nos USA, e caso um dia eu seja obrigado a trazer os recursos para o Brasil (algo que eu não pretendo), precisarei fazer um bom planejamento tributário.

**Reply by lima-emanuel** [reply]
Então, essa sempre foi uma dúvida minha. Só se paga o imposto se nacionalizar a renda? Acho que não né. Acho que tem que declarar o ganho de renda em dólares australianos, mesmo que eles nunca cheguem no Brasil, não?

**Reply by raulfragoso** [reply]
Do ponto de vista da empresa, meu contador diz que não, pois somente os recursos enviados ao Brasil devem ser declarados ao faturar a prestação de serviço. Minha intenção é manter 20% nos USA e nacionalizar os 80% restantes.

Sobre a compra de stocks nos USA, eu tenho que declarar os ativos na minha declaração anual de IRPF, já os dividendos (se houverem) são retidos na fonte pagadora nos USA.

**Reply by copercini** [reply]
Eu acho que precisa pagar impostos mesmo não trazendo o dinheiro para o Brasil, pois se não era só deixar tudo fora e não precisaria pagar impostos =)

O problema é que é tão nebuloso e burocrático que seu contador aparentemente resolveu ir pelo caminho mais fácil e isso pode te gerar problemas na hora que vc colocar os investimentos do exterior na declaração anual de IRPF, pois o dinheiro "brotou" lá e não foi enviado pela sua empresa através de lucros/dividendos ou pró-labore

**Reply by raulfragoso** [reply]
Pois é, eu confesso que também fico com dúvidas, mas provavelmente eu teria problemas ao faturar o valor total em dólar enquanto apenas parte do valor efetivamente entra na conta em reais da empresa no Brasil. Portanto a opção "mais correta" seria mesmo nacionalizar todos os recursos em AUD para BRL, e só depois enviar de BRL para USD no exterior. O grande problema, obviamente, é que essa opção é a mais onerosa, não apenas do ponto de vista tributário, mas principalmente com o spread em duplicidade e taxas de IOF.

Estou pensando se há uma maneira de resolver esse impasse com uma medida simples, por exemplo pedindo ao pagador na Australia para que envie diretamente parte dos recursos para minha conta na TD Ameritrade.

**Reply by samiraguiar** [reply]
@raulfragoso eu concordo com o @copercini, mas não creio que seja necessário nacionalizar os valores como você comentou, desde que eles sejam corretamente declarados pela PJ (com pagamento dos impostos referente ao recebimento), e posteriormente pela PF na DIRPF.

Achei esse artigo aqui que esclarece um pouco a questão:
https://bueno.tax/artigos/2021/01/distribuicao-de-dividendos-sem-cobertura-cambial-aspectos-regulatorios-e-tributarios-da-transferencia-de-valores-detidos-no-exterior-por-pessoa-juridica-brasileira-para-conta-bancaria-estrangeira-de/

> Considerando o exposto acima, pode-se concluir que o pagamento de dividendos de conta bancária no exterior, detida pela Sociedade, para a conta bancária no exterior, detida pela Pessoa Física, esta última sua sócia, sem o ingresso financeiro ou repatriação cambial (efetiva ou simbólica) de tais valores ao Brasil, não é proibido pela legislação atualmente em vigor.

**Reply by raulfragoso** [reply]
@samiraguiar obrigado pelo link, muito elucidativo! Vou encaminhar ao meu contador.
Se entendi bem, posso manter a transferência dos recursos diretamente para a conta nos USA, desde que a PJ recolha o IRPJ e CSLL do valor total e declare a transferência para a PF no exterior como distribuição de lucros. Minha única dúvida é como ficaria a emissão da NF eletrônica de serviços, visto que o faturamento é baseado no valor nacionalizado, e serve também como base de cálculo para o recolhimento dos impostos da PJ no Brasil.

**Reply by samiraguiar** [reply]
@raulfragoso a emissão fica igual, você só converte usando a cotação oficial do Banco Central usando a última cotação do dia do pagamento:
> Na nota fiscal internacional, os valores devem ser discriminados em reais e convertidos conforme o câmbio do dia do pagamento. É importante fazer a emissão da nota fiscal na mesma data em que o pagamento for realizado. Assim, dá para inserir o valor correto, de acordo com a atualização cambial. Caso não seja possível, lembre-se de verificar o valor do câmbio na data do recebimento.
> https://www.remessaonline.com.br/blog/nota-fiscal-exportacao-de-servicos-como-emitir/

e também
> Ou seja, se você receber US$ 10.000,00 (Dez mil dólares) e na data o câmbio do dólar estiver em R$ 3,25, seu faturamento será calculado da seguinte forma: US$ 10.000,00 x R$ 3,25 = R$ 35.250,00.
> 
> Se contabiliza a variação cambial até o dia do recebimento pelo serviço, mas o cálculo de faturamento será feito com base no dia que foi feito o negócio.
> https://www.jornalcontabil.com.br/invoice-internacional-como-emitir-notas-fiscais-para-o-exterior/

**Reply by raulfragoso** [reply]
Eu tenho emitido a NF mensalmente com o valor exato já em reais depois do recebimento através da Payoneer. Isso facilita bastante a contabilidade, pois tenho precisão exata ao comparar o faturamento, escrituração contábil e também nos extratos bancários.
Verei com o contador como fica a partir de agora.

**Reply by mcferdev** [reply]
@raulfragoso Eu fui abrir uma conta na TD Ameritrade, mas eles pedem um extrato bancario da conta. Por acaso seria a da Payoneer ? Porque se for, nao vai bater com os dados da TD Ameritrade. Voce enviou de sua conta PF normal do BR ? E eles aceitaram transferencia da Payoneer por qual meio? Abs.

**Reply by raulfragoso** [reply]
> @raulfragoso Eu fui abrir uma conta na TD Ameritrade, mas eles pedem um extrato bancario da conta. Por acaso seria a da Payoneer ? Porque se for, nao vai bater com os dados da TD Ameritrade. Voce enviou de sua conta PF normal do BR ? E eles aceitaram transferencia da Payoneer por qual meio? Abs.

Sim, eu enviei da minha conta pessoa-física mesmo (Nubank e Inter). Pedem uma conta utilitária (energia, telefone, etc) também, eu enviei a minha conta de energia. Enviei tudo por fax junto com o formulário. A transferência pela Payoneer é feita como pagamento, através da opção "Pay -> Pay to a recipient's bank account". É necessário cadastrar sua conta da TD Ameritrade primeiro na Payoneer.

**Reply by mcferdev** [reply]
Show. Vou testar...minha conta PF é Bradesco... preciso ver como ter esse extrato em ingles, porque é em portugues e eles pedem ingles. Tu traduziu ou tinha em ingles?
Essas transfs da Payoneer sempre mordem 2%. Uma lástima :(

Obrigado.

**Reply by raulfragoso** [reply]
Enviei o original, em português mesmo.  Esses 2% são uma lástima mesmo, mas é o que é, não há o que fazer.

**Reply by danisteffen** [reply]
@raulfragoso , suas explicações são excelentes.
Eu tenho empresa no Brasil e, de acordo com a Contabilivre (nhé), eu tenho que receber no Brasil e depois enviar para os EUA como distribuição de lucros. É um inferno, IOF 2x, tarifas mil de bancos, etc. 
Tenho Payoneer da minha empresa e queria manter a parte da distribuição de lucros lá, sem nacionalizar e depois enviar de volta para os EUA, economizando em IOF, tarifas e tempo.
Obviamente que a Contabilivre não sabe nada sobre como fazer isso.
Minha pergunta é como você lança a movimentação no Payoneer na sua contabilidade? Agradeço muitíssimo se você puder me explicar.
Danielle
